Cabal/src/Distribution/Simple/Utils.hs

```diff
@@ -1198,7 +1198,7 @@ findFileCwd verbosity cwd searchPath fileName =
   findFirstFile
     (cwd </>)
     [ path </> fileName
-    | path <- nub searchPath
+    | path <- ordNub searchPath
```

Comment:
Do you have a reason behind this change? (and the ones below). I am sure you do but I don't see it written anywhere.

Reply:
This is an orthogonal change. I can drop the commit.

It was just an improvement on the fly, as I was passing by.

`ordNub` leverages the fact that the type it is applied to is `Ord`, rather than `nub` which only uses the `Eq` instance. So `ordNub` has a lower algorithmic complexity than `nub` which is O(n^2).